Adamant/Modules/Settings/Notifications/NotificationsViewModel.swift

```diff
@@ -18,40 +18,42 @@ final class NotificationsViewModel: ObservableObject {
     @Published var notificationsMode: NotificationsMode = .disabled
     @Published var notificationSound: NotificationSound = .inputDefault
     @Published var notificationReactionSound: NotificationSound = .none
-    @Published var githubRowImage: UIImage = .asset(named: "row_github") ?? UIImage()
-    @Published var notificationsTitle: String = .localized("SecurityPage.Row.Notifications")
     @Published var presentSoundsPicker: Bool = false
     @Published var presentReactionSoundsPicker: Bool = false
     @Published var openSafariURL: Bool = false
     @Published var inAppSounds: Bool = false
     @Published var inAppVibrate: Bool = true
     @Published var inAppToasts: Bool = true
     
-    var safariURL = URL(string: "https://github.com/Adamant-im")!
+    let notificationsTitle: String = .localized("SecurityPage.Row.Notifications")
+    let safariURL = URL(string: "https://github.com/Adamant-im")!
+    let githubRowImage: UIImage = .asset(named: "row_github") ?? UIImage()
```

Comment:
minor: `githubRowImage` is only used in the View, so let’s move it here as a constant (at the bottom of the file)